usage/rest-server/src/test/java/brooklyn/rest/HaHotCheckTest.java

```diff
@@ -43,75 +43,61 @@
 
 public class HaHotCheckTest extends BrooklynRestResourceTest {
 
-    private ManagementContextMock mgmtMock;
-
-    @Override
-    @BeforeClass(alwaysRun = true)
-    public void setUp() throws Exception {
-        mgmtMock = new ManagementContextMock();
-        super.setUp();
-    }
-
-    @Override
-    @AfterClass(alwaysRun = true)
-    public void tearDown() throws Exception {
-        super.tearDown();
-    }
-
     @BeforeMethod(alwaysRun = true)
-    public void setUpMethod() {
-        mgmtMock.setState(ManagementNodeState.MASTER);
-    }
+    public void setUp() throws Exception { super.setUp(); }
 
+    @AfterMethod(alwaysRun = false)
+    public void tearDown() throws Exception { super.tearDown(); }
+    
     @Override
     protected void addBrooklynResources() {
-        config.getSingletons().add(new ManagementContextProvider(mgmtMock));
```

Comment:
Probably can do the same for the other usages of `ManagementContextProvider`  and get rid of it altogether (not really important).